README.md

```diff
@@ -83,7 +83,8 @@ Optional:
 **Do *not* use the zip file from github: do a recursive clone only**
 
 ```bash
-$ git clone --recursive https://github.com/monero-project/kovri
+$ git clone --recursive https://github.com/monero-project/kovri.git
```

Comment:
>kovri.git

Thought technically ok, it's not necessary with github.

Reply:
Sure, I know it's redundant but I'm so used to seeing the extension written out that I added it without thinking too much. Looking through the monero-project repos, I see it written out in monero-gui but not in monero. The URL you get from the GitHub "Clone or download button" does contain the .git extension.

As I'll try to modify this PR anyway, you think it's better w/ or w/o the .git extension?

Reply:
Preferably without but it's not a huge issue.